examples/AsyncSend/AsyncSend.ino

```diff
@@ -34,7 +34,7 @@ Buffer received - next send in 2 sec
   And in the remote terminal 3072 characters sent [.........  ...........] and so on.
 */
 
-#include <Arduino.h>
+#include <HardwareSerial.h>
```

Comment:
neither include is needed if this in an `ino` file that will be compiled with Arduino. Arduino.h is automatically included. If this is compiled with PIO, then please keep Arduino.h for compatibility. Same for the other examples

Reply:
Ok! Didn't know that! Goal was to limit the inclusions to the bare minimum.